gen3git.py

```diff
@@ -261,19 +261,19 @@ def main(args=None):
             print("Cannot find tag: %s" % to_tag)
             return
     else:
-        stop_commit = git.head.commit
+        stop_commit = repo.get_commits().__getitem__(0)  # latest commit
         if hasattr(args, "new_tag"):
             stop_tag = args.new_tag
         else:
-            for tag in git.tags:
-                if tag.commit.hexsha == stop_commit.hexsha:
+            for tag in repo.get_tags():
+                if tag.commit.sha == stop_commit.sha:
                     stop_tag = tag.name
-    repo.get_commit(stop_commit.hexsha)
-    print("Generate changelog up to commit: %s" % stop_commit.hexsha)
+    repo.get_commit(stop_commit.sha)
+    print("Generate changelog up to commit: %s" % stop_commit.sha)
```

Comment:
I didn't find doc to `Commit.sha` 🤔  this is the closest: https://gitpython.readthedocs.io/en/stable/reference.html#git.objects.base.Object.binsha

What is it and what is the difference than `hexsha`?

Reply:
the `gitpython` lib uses `hexsha`, the `pygithub` lib uses `sha` (https://pygithub.readthedocs.io/en/latest/github_objects/Commit.html#github.Commit.Commit.sha)
the `git` object and the `repo` object are from different libs 🤣

Reply:
😱